episodes/contact-matrices.Rmd

```diff
@@ -0,0 +1,271 @@
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+
+
+## Analyses with contact matrices
+
+Contact matrices can be used in a wide range of epidemiological analyses, they can be used:
+
++ to calculate the basic reproduction number while accounting for different rates of contacts between age groups [(Funk et al. 2019)](https://doi.org/10.1186/s12916-019-1413-7),
++ to calculate final size of an epidemic, as in the R package `{finalsize}`,
++ to assess the impact of interventions finding the relative change between pre and post intervention contact matrices to calculate the relative difference in $R_0$ [(Jarvis et al. 2020)](https://doi.org/10.1186/s12916-020-01597-8),
++ and in mathematical models of tranmission within a population, to account for group specific contact patterns.
+
+
+However, all of these applications require us to perform some additional calculations using the contact matrix. Specifically, there are two main calculations we often need to do:
+
+1. **Convert contact matrix into expected number of secondary cases**
+
+If contacts vary between groups, then the average number of secondary cases won't be equal simply to the average number of contacts multiplied by the probability of transmission-per-contact. This is because the average amount of transmission in each generation of infection isn't just a matter of whom a group came into contact with; it's about whom *their contacts* subsequently come into contact with. The function `r_eff` in the package `{finalsize}` can perform this conversion, taking a contact matrix, demography and proportion susceptible and converting it into an estimate of the average number of secondary cases generated by a typical infectious individual (i.e. the effective reproduction number).
+
+2. **Convert contact matrix into contact rates**
+
+Whereas a contact matrix gives the average number of contacts that one groups makes with another, epidemic dynamics in different groups depend on the rate at which one group infects another. We therefore need to scale the rate of interaction between different groups (i.e. the number of contacts per unit time) to get the rate of transmission. However, we need to be careful that we are defining transmission to and from each group correctly in any model. Specifically, the entry $(i,j)$ in a mathematical model contact matrix represents contacts by group $i$ made with group $j$. But if we want to know the rate at which a group $i$ are getting infected, then we want to calculate the proportion of individuals infected in each group $j$ ($I_j/N_j$), multiplied by the number of contacts made with group $i$ ($C[i,j]$), multiplied by the transmission risk per contact ($\beta$).
```

Comment:
Something is flipped between
- "the entry $(i,j)$ in a mathematical model contact matrix represents contacts by group $i$ made with group $j$"
- "the number of contacts made with group $i$ ($C[i,j]$)"
In an attempt to phrase this clearer (acknowledging how hard it is to communicate this):

But if we want to know the rate at which a group $i$ are getting infected, then we want to calculate the number of contacts in group $j$ made by susceptibles in group $i$ ($S_i C[i,j]$) with the proportion of those contacts that are infectious ($I_j/N_j$), and the transmission risk per contact ($\beta$).

Reply:
You probably noticed this but re-reading it this should probably be "calculate" -> "multiply"